packages/pf3-component-mapper/src/form-fields/button.js

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Button } from 'patternfly-react';
 
-const ButtonOverride = (props) => <Button style={{ marginLeft: 3 }} { ...props } />;
+export const Button2 = ({ label, variant, ...rest }) => {
```

Comment:
Maybe keep the original name :smile: i don't like Button2. Or import the PF button as an alias 
```jsx
import {Button as PfButton} from 'patternfly-react'
```

Reply:
Maybe ButtonTheSecond? :D